var/spack/repos/builtin/packages/tfel/package.py

```diff
@@ -45,8 +45,10 @@ class Tfel(CMakePackage):
     version("rliv-1.2", branch="rliv-1.2")
 
     # released version
+    # 4.0.0 fails 10930:crossed2deltachaboche_mtest not matching expected result value:
```

Comment:
This test works on every computer I tried (including all recent versions of Ubuntu) and a lot of compilers. I would prefer to hold 4.0 as the default as I need feedbacks on this version as soon as possible.

Reply:
OK, I backed this change out of the new push, if reproduce it with another compiler, I'll report it to with the details.